packages/client/modules/teamDashboard/components/UnpaidTeamModal/UnpaidTeamModal.tsx

```diff
@@ -54,6 +69,50 @@ const UnpaidTeamModal = (props: Props) => {
   const {team} = viewer
   if (!team) return null
   const {name: teamName, organization, lockMessageHTML} = team
+
+  const {name: organizationName} = organization
+
+  const {id: orgId, billingLeaders, name: orgName} = organization
+  const [firstBillingLeader] = billingLeaders
+  const billingLeaderName = firstBillingLeader?.preferredName ?? 'Unknown'
+  const email = firstBillingLeader?.email ?? 'Unknown'
+  const isALeader = billingLeaders.findIndex((leader) => leader.id === viewerId) !== -1
+
+  const goToBilling = (upgradeCTALocation: UpgradeCTALocationEnum) => {
+    SendClientSegmentEventMutation(atmosphere, 'Upgrade CTA Clicked', {
+      upgradeCTALocation
+    })
+    history.push(`/me/organizations/${orgId}`)
+  }
+
+  if (organization.lockedAt) {
+    return (
+      <DashModal>
+        <DialogTitle>{'Organization Locked'}</DialogTitle>
+        <LockedAtContent>
+          Unfortunately, <strong>{organizationName}</strong> has exceeded the{' '}
+          {Threshold.MAX_PERSONAL_TIER_TEAMS} teams limit on the Starter Plan for more than{' '}
+          {Threshold.PERSONAL_TIER_LOCK_AFTER_DAYS} days, and your account has been locked.
+          <br />
+          <br />
```

Comment:
+1 having 2 `<br />` looks like we should put the blocks in `<p>` and style it accordingly